charts/kong/templates/NOTES.txt

```diff
@@ -26,3 +26,17 @@ Kong: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/gettin
 {{- end -}}
 
 {{- include "kong.deprecation-warnings" $warnings -}}
+
+{{- if contains "LoadBalancer" .Values.proxy.type -}}
```

Comment:
```suggestion
{{- if .Values.demo -}}
```
Shouldn't be like that to leverage the added demo toggle? 🤔

Reply:
🤦 This is why we have PR reviews!